FFXIVClientStructs/FFXIV/Client/System/Framework/Framework.cs

```diff
@@ -18,12 +20,19 @@ public unsafe partial struct Framework {
 
     [FieldOffset(0x570)] public SavedAppearanceManager* SavedAppearanceData;
 
+    [FieldOffset(0x1670)] public NetworkModuleProxy* NetworkModuleProxy;
     [FieldOffset(0x1678)] public bool IsNetworkModuleInitialized;
     [FieldOffset(0x1679)] public bool EnableNetworking;
-    [FieldOffset(0x1680)] public long ServerTime;
+    [FieldOffset(0x1680)] public long ServerTime;  // TODO: change to uint
+    [FieldOffset(0x1688)] public long PerformanceCounterInMilliSeconds;
+    [FieldOffset(0x1688)] public long PerformanceCounterInMicroSeconds;
+    [FieldOffset(0x1698)] public uint TimerResolutionMillis;
+    [FieldOffset(0x16A0)] public long PerformanceCounterFrequency;
+    [FieldOffset(0x16A8)] public long PerformanceCounterValue;
     [FieldOffset(0x16F8)] public TaskManager TaskManager;
     [FieldOffset(0x16B8)] public float FrameDeltaTime;
     [FieldOffset(0x16C8)] public uint FrameCounter;
+    [FieldOffset(0x1768)] public ClientTime ClientTime;
     [FieldOffset(0x1770)] public long EorzeaTime;
     [FieldOffset(0x1798)] public long EorzeaTimeOverride;
     [FieldOffset(0x17A0)] public bool IsEorzeaTimeOverridden;
```

Comment:
```suggestion
    [FieldOffset(0x1768)] public ClientTime ClientTime;
    [FieldOffset(0x1770), Obsolete("Use ClientTime.EorzeaTime")] public long EorzeaTime;
    [FieldOffset(0x1798), Obsolete("Use ClientTime.EorzeaTimeOverride")] public long EorzeaTimeOverride;
    [FieldOffset(0x17A0), Obsolete("Use ClientTime.IsEorzeaTimeOverridden")] public bool IsEorzeaTimeOverridden;
```

Reply:
Was on it,just ttok me a bit as this resolved some fields/getters

Reply:
🤷‍♂️ 😄
After seeing your commit, `EorzeaTimeRemainder` should've been named `EorzeaTimeMilliseconds` (in `ClientTime`).
Also, you have typod `NetworConfig` in the comment in `LobbyUIClient`. 🙈